source/MaterialXView/Viewer.h

```diff
@@ -357,11 +394,14 @@ class Viewer : public ng::Screen
     unsigned int _wedgeImageCount;
 
     // Texture baking
-    bool _bakeTextures;
-    bool _bakeHdr;
-    bool _bakeAverage;
-    bool _bakeOptimize;
-    bool _bakeRequested;
+    bool _bakeTextures = false;
```

Comment:
Following the pattern for other member variables of Viewer, these should be initialized in the constructor rather than the class declaration.